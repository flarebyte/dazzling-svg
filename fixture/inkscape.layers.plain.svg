<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->

<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   version="1.1"
   width="744.09448"
   height="1052.3622"
   id="svg2">
  <defs
     id="defs4" />
  <metadata
     id="metadata7">
    <rdf:RDF>
      <cc:Work
         rdf:about="">
        <dc:format>image/svg+xml</dc:format>
        <dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
        <dc:title></dc:title>
      </cc:Work>
    </rdf:RDF>
  </metadata>
  <g
     id="layer1" />
  <g
     id="layer2">
    <rect
       width="185.71428"
       height="125.71429"
       x="128.57143"
       y="178.07646"
       id="rect2988"
       style="fill:#4e9a06" />
    <g
       id="g2994">
      <rect
         width="320"
         height="108.57143"
         x="217.14285"
         y="378.07648"
         id="rect2990"
         style="fill:#3465a4" />
      <path
         d="m 474.2857,513.79077 a 72.85714,67.14286 0 1 1 -145.71428,0 72.85714,67.14286 0 1 1 145.71428,0 z"
         transform="translate(-25.714286,-17.142857)"
         id="path2992"
         style="fill:#3465a4" />
    </g>
  </g>
  <g
     id="layer3">
    <g
       id="g3002">
      <path
         d="m 328.57143,682.36218 a 101.42857,84.285713 0 1 1 -202.85715,0 101.42857,84.285713 0 1 1 202.85715,0 z"
         id="path2998"
         style="fill:#ce5c00" />
      <path
         d="m 420.00001,788.07648 a 94.285713,81.428574 0 1 1 -188.57143,0 94.285713,81.428574 0 1 1 188.57143,0 z"
         id="path3000"
         style="fill:#ce5c00" />
    </g>
  </g>
  <g
     id="layer4">
    <rect
       width="337.14285"
       height="88.571426"
       x="85.714287"
       y="66.647896"
       id="rect3006"
       style="fill:#a40000" />
  </g>
</svg>
