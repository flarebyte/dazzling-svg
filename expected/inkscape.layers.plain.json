{
  "shapes": [
    {
      "fill": "#4e9a06",
      "d": "M128.571 178.076H314.28499999999997V303.78999999999996H128.571z",
      "Type": "path"
    },
    {
      "d": "M217.143 378.076H537.143V486.64700000000005H217.143z",
      "Type": "path"
    },
    {
      "d": "m 474.2857,513.79077 a 72.85714,67.14286 0 1 1 -145.71428,0 72.85714,67.14286 0 1 1 145.71428,0 z",
      "transform": "translate(-25.714286,-17.142857)",
      "Type": "path"
    },
    {
      "d": "m 328.57143,682.36218 a 101.42857,84.285713 0 1 1 -202.85715,0 101.42857,84.285713 0 1 1 202.85715,0 z",
      "Type": "path"
    },
    {
      "d": "m 420.00001,788.07648 a 94.285713,81.428574 0 1 1 -188.57143,0 94.285713,81.428574 0 1 1 188.57143,0 z",
      "Type": "path"
    },
    {
      "fill": "#a40000",
      "d": "M85.714 66.648H422.85699999999997V155.219H85.714z",
      "Type": "path"
    }
  ]
}